From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Fri, 20 May 2016 08:46:11 +0200
Subject: [PATCH] bfd: switch to release mode

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 bfd/development.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bfd/development.sh b/bfd/development.sh
index 6bbef6dd744d..0e65c0788277 100644
--- a/bfd/development.sh
+++ b/bfd/development.sh
@@ -16,4 +16,4 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 # Controls whether to enable development-mode features by default.
-development=true
+development=false
