Index: glibc-2.7/README
===================================================================
--- glibc-2.7.orig/README	2008-03-10 18:38:00.000000000 +0100
+++ glibc-2.7/README	2008-03-10 18:38:19.000000000 +0100
@@ -91,3 +91,5 @@
 The GNU C Library is free software.  See the file COPYING.LIB for copying
 conditions, and LICENSES for notices about a few contributions that require
 these additional notices to be distributed.
+
+-- Coldfire port for ptxdist 2008.03.10
