Index: glibc-2.5/configure
===================================================================
--- glibc-2.5.orig/configure	2008-03-08 10:28:12.000000000 +0100
+++ glibc-2.5/configure	2008-03-08 10:28:18.000000000 +0100
@@ -4229,7 +4229,7 @@
   echo "$as_me:$LINENO: result: $ac_prog_version" >&5
 echo "${ECHO_T}$ac_prog_version" >&6
 fi
-if test $ac_verc_fail = yes; then
+if test $ac_verc_fail = yes -a -z "$enable_hacker_mode"; then
   critic_missing="$critic_missing gcc"
 fi
 
