Index: glibc-2.5/sysdeps/unix/sysv/linux/getcwd.c
===================================================================
--- glibc-2.5.orig/sysdeps/unix/sysv/linux/getcwd.c
+++ glibc-2.5/sysdeps/unix/sysv/linux/getcwd.c
@@ -38,6 +38,9 @@
 # define NO_ALLOCATION	1
 #endif
 
+#ifndef MAX
+# define MAX(a, b) ((a) < (b) ? (b) : (a))
+#endif
 
 #if __ASSUME_GETCWD_SYSCALL > 0
 /* Kernel 2.1.92 introduced a third way to get the current working
