---
 libiberty/Makefile.in |    1 +
 1 file changed, 1 insertion(+)

Index: binutils-2.21/libiberty/Makefile.in
===================================================================
--- binutils-2.21.orig/libiberty/Makefile.in
+++ binutils-2.21/libiberty/Makefile.in
@@ -246,6 +246,7 @@ $(TARGETLIB): $(REQUIRED_OFILES) $(EXTRA
 	  $(AR) $(AR_FLAGS) $(TARGETLIB) \
 	    $(REQUIRED_OFILES) $(EXTRA_OFILES) $(LIBOBJS); \
 	  $(RANLIB) $(TARGETLIB); \
+	  cp $(TARGETLIB) ../ ; \
 	  cd ..; \
 	else true; fi
 
